src/stores/RedisStore.ts

```diff
@@ -0,0 +1,40 @@
+import { connect, RedisConnectOptions } from "https://deno.land/x/redis@v0.25.0/mod.ts"
+import type { Ratelimit } from "../types/types.d.ts";
+import { Store } from "./AbstractStore.ts";
+
+export class RedisStore extends Store {
+    private readonly store;
+
+    constructor(options: RedisConnectOptions) {
+        super();
+        this.store = connect({
+            hostname: options.hostname,
+            port: options.port,
+            tls: options.tls,
+            db: options.db,
+            password: options.password,
+            name: options.name,
+            maxRetryCount: options.maxRetryCount,
+            retryInterval: options.retryInterval,
+        })
+    }
+
+    public async get(ip: string) {
+        const data = await (await this.store).get(ip)
```

Comment:
It only needs one `await` and the `await` doesn't need to be inside `()` remove `()`.

Reply:
![image](https://user-images.githubusercontent.com/59208097/147293165-bb7ebf29-7e1b-4104-8483-0bc0d48eec3e.png)
when i remove the brackets this error comes up